### ISSUE TITLE
Reviving cleanco

### ISSUE THREAD

**Issue by wangrat** [psolin/cleanco#91]
Hi,

I'm interested in contributing to this project, but I noticed that there are 3 PRs that have been waiting for a review for a long time. I understand that the maintainers might not have the time or interest to continue working on the project, but I'm wondering if it would be possible to grant write permissions to new contributors who are willing to help maintain and improve cleanco.

If this isn't feasible, an alternative approach could be to fork the project and publish it on PyPI under a new name, such as "cleanco-v2" or "py-cleanco". However, this would require migrating the existing issues, reaching out to PR authors, and potentially contacting past forkers who have built upon this package. I would much prefer to continue working on the original cleanco project if possible.

I'm open to discussing the best way to move forward and ensure the project remains active and well-maintained. Please let me know your thoughts.

Thank you for your consideration.

**Comment by batlle-mon** [comment]
I second the sentiment, I'm interested to contribute to the project as well.

**Comment by wangrat** [comment]
@petri @psolin

**Comment by petri** [comment]
Do you have something specific you need or would like to add? If there is just something in particular you need, please file an issue and a PR.

Or do you use cleanco continuously in your work and would like to contribute on an on-going bases as one of the maintainers because of that?

**Comment by wangrat** [comment]
Hello @petri,

Sorry for the late reply!

I'm currently using cleanco for a project that involves parsing company names from around the world, and it's really important for me that it works perfectly. I was ready to dive in and start fixing things based on what I've seen and the existing issues others have raised. But, when I saw a bunch of old PRs sitting unreviewed, I got a bit worried about spending time on contributions that might not get attention.

However, I've noticed some of those PRs have been merged since I first reached out, and that's really reassuring! It looks like things are moving again, which is awesome. I'm all set to start contributing as planned.

Also, I really want to say thanks to you and everyone else keeping cleanco going. I genuinely appreciate all the work you've put in—I hope it doesn’t seem like I'm taking that for granted.

Looking forward to getting involved and helping out where I can.

Cheers,
Adam
